docs/guide/v2_upgrade_guide.md

```diff
@@ -246,3 +246,47 @@ val entity = dyDatabase.dyTable.dyEntitiy.load(
 + consistentReads = true
 )
 ```
+
+## Changes to Misk
+
+If you are using [Misk](https://github.com/cashapp/misk) you will need to update your dependencies and DI configuration
+
+### Dependencies
+
+```diff
+-val miskAwsDynamodb = "com.squareup.misk:misk-aws-dynamodb:VERSION"
+-val miskAwsDynamodbTesting = "com.squareup.misk:misk-aws-dynamodb-testing:VERSION"
++val miskAws2Dynamodb = "com.squareup.misk:misk-aws2-dynamodb:VERSION"
++val miskAws2DynamodbTesting = "com.squareup.misk:misk-aws2-dynamodb-testing:VERSION"
+```
+
+### DynamoDbModule
+
+
+```diff
```

Comment:
TIL that markdown supports diffs

Reply:
Here is an [incomplete list of support languages](http://dontcry2013.github.io/2019/12/02/github-code-block/), it has some oddballs like `diff` and `profile` which are exactly _languages._

The [complete list](https://github.com/github/linguist/blob/master/vendor/README.md) very unhelpfully does not include the short names that you need to provide to the backticks,.

Reply:
If I'm following correctly, I think it'd be better to have an AWS config update section and (if also necessary) a Misk DI section. The AWS retry policy change is also something worth highlighting since it is easy to mess up (we did!)